test/integration/targets/vmware_vswitch/aliases

```diff
@@ -0,0 +1,3 @@
+posix/ci/cloud/vcenter
+cloud/vcenter
+skip/python3
```

Comment:
What's needed to make this support Python3?
Generally speaking we shouldn't be adding `skip/python3` in, we need to fix the issues